### ISSUE TITLE
crossdomain.xml errors

### ISSUE THREAD

**Issue by maxwondercorn** [felixrieseberg/ember-electron#251]
I’m changing this post to be more specific and it should probably be retitled "Can not find module in package or something similar.  Hopefully changing the original post is not breaking a GitHub norm.

My project is complete and runs without issue using _ember electron_. When I package the applicaiton (ember electron:package) I have a native module that is not being included  generating a "Can't find module error when launching the package.

Specifically, I’m using Linvodb3 to persist data to disk.  It’s the only node module, other than electron that uses node.js require() to load.  The error seems to indicate that it’s looking to resolve inside the generated asar file?

The code snippet below shows the require in the adapter.

app/adapters/application.js

    import Ember from 'ember';
    import DS from 'ember-data';
    const LinvoDB = require("linvodb3");

    const { run } = Ember;

    LinvoDB.defaults.store = { db: require("leveldown") }

    export default DS.Adapter.extend({
        filename: '../icare.db',

	// adapter code here
    }

While the app breaks on load, I’m assuming that I may also have an issue with the Leveldown require when this issue is resolved.  I did try window.requireNode and that did not make a difference

In the original post, I thought the failure of compiling crossdomain.xml (similar to #240) was aborting the package process.  After reviewing it, I see that the process is completing.  It just doesn’t include or link properly to the Linvodb3 module.

**Comment by benoror** [comment]
@maxwondercorn maybe you're having a similar issue: https://github.com/felixrieseberg/ember-electron/issues/272

Did you find a workaround?

**Comment by pichfl** [comment]
There might be some problem with using `require()` to call node modules. Have you tried using `requireNode()` as described in the [Ember-Electron documentation?](https://github.com/felixrieseberg/ember-electron/blob/master/docs/guides/usage.md#conflict-between-ember-and-electron-require)

**Comment by bendemboski** [comment]
@maxwondercorn are you sure your dependency is a `dependency` and not a `devDependency`? Can you run `ember electron:assemble` and then look in `electron-out/project/node_modules` to see if it's present?

**Comment by maxwondercorn** [comment]
Thanks for input.  I finally got back to this a couple days ago.  I originally did have my dependency in devDepencdecy.  After moving it, my problem was still there and that's when I changed my post.  electron:package would skip the Preparing native Dependencies step, packaging without the dependencies.  In the end, I created test app and it worked. I then rebuilt the app from the ground up and it worked.  I can't say why the rebuilt app worked - the package.json files matched  I am wondering if I don't have some type of issue with my dev computer.  When I packaged the app on a Win10 machine, I tried the original version and it worked.  I consider this issue closed, even without understating the resolution.  BTW, I do love this add-on
